src/layouts/DefaultLayout.vue

```diff
@@ -50,6 +50,19 @@
             </v-list-item>
           </v-list>
         </v-menu>
+        <v-list density="compact" v-if="navigationMenuComponents.length === 1">
```

Comment:
Should this still be a `v-list` and should it look different than the style used for the menu button above?:
```vue
            <v-list-item
              aria-label="Menu button"
              v-bind="props"
              class="ma-1"
              rounded
              variant="tonal"
            >
```

Reply:
I assumed so, yes